src/main/steps/tasklistresponse/utils.ts

```diff
@@ -32,10 +39,19 @@ export const mapDataInSession = (userCase: CaseWithId, userId: UserDetails['id']
       Object.assign(userCase, mapSupportYouNeedDetails(partyDetails));
     }
 
-    if (partyDetails.response?.keepDetailsPrivate?.confidentiality) {
-      Object.assign(userCase, mapKeepYourDetailsPrivate(partyDetails));
+    if (partyDetails.response.consent) {
+      Object.assign(userCase, mapConsentToApplicationDetails(respondentDetails));
```

Comment:
change the mappers to accept partydetails

Reply:
Done